pkg/yurtctl/cmd/yurttest/kindinit/init.go

```diff
@@ -30,11 +30,11 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog/v2"
 
-	"github.com/openyurtio/openyurt/pkg/yurtctl/constants"
-	enutil "github.com/openyurtio/openyurt/pkg/yurtctl/util/edgenode"
-	kubeutil "github.com/openyurtio/openyurt/pkg/yurtctl/util/kubernetes"
-	strutil "github.com/openyurtio/openyurt/pkg/yurtctl/util/strings"
-	tmplutil "github.com/openyurtio/openyurt/pkg/yurtctl/util/templates"
+	"github.com/openyurtio/openyurt/pkg/yurtadm/constants"
```

Comment:
We should import `pkg/yurtctl/constants` instead of `pkg/yurtadm/constants`. Any constants used in this file should be in the `pkg/yurtctl/constants`, even there's duplication. In this way, we can decouple yurtctl with yurtadm.